yaptide/celery/utils/pymc.py

```diff
@@ -286,7 +287,7 @@ def read_file(event: threading.Event,
 
     logging.info("Parsing log file for task %d finished", task_id)
     up_dict = {
-        "simulated_primaries": requested_primaries,
+        "simulated_primaries": simulated_primaries,
```

Comment:
Lets imagine we have simulation with 1000 particles.

It may happen that simulation finished, but the `simulated_primaries` which was extracted periodically from the log file stayed at for example 990.

Here the intention is to enforce that once we know the simulation is completed, to set that all the particles were simulated.

Take a look here: 
https://github.com/yaptide/yaptide/blob/882e62c0c24b42bd509173e730da175522c919f2/yaptide/batch/watcher.py#L102-L104